.gitignore

```diff
@@ -4,4 +4,4 @@ elasticsearch_exporter
 *-stamp
 .tarballs
 /vendor
-.idea
+vendor/
```

Comment:
This seems unrelated.

Reply:
to keep consistent with `.gitignore` of the main branch